QA_Nano.md

```diff
@@ -0,0 +1,107 @@
+# Create Nano Contract
+This test suite will execute the full flow of the [Bet blueprint](https://explorer.alpha.nano-testnet.hathor.network/blueprint/detail/3cb032600bdf7db784800e4ea911b10676fa2f67591f82bb62628c234e771595), ensuring all interface interaction is validated with its error handling cases.
+
+More about Nano Contracts on our [official docs](https://docs.hathor.network/explanations/features/nano-contracts/).
+
+# Initializing connection
+- Open the Desktop Wallet with an existing seed
+- Go to "Settings" and change its server to `https://hathorplay.nano-testnet.hathor.network/v1a/`
+- Go to "Settings" and change its mining server to `https://txmining.nano-testnet.hathor.network/`
+- Activate the `nano-contracts-desktop.rollout` feature toggle on unleash for this wallet's unique id
+- Wait a while for the "Nano Contract" tab to appear at the navigation header
+
+# Blueprint Selection
+- Navigate to the "Nano Contract" tab
+- Click the "Create a nano contract" button
+- Write nothing and press "Fetch": the field should turn red
+- Write a single letter and press "Fetch": there should be an error fetching blueprint information.
+- Write the `Bet` blueprint id (`3cb032600bdf7db784800e4ea911b10676fa2f67591f82bb62628c234e771595`) and press "Fetch"
+- The blueprint data should be displayed
+- Click "Select New Blueprint": the screen and its input should now be empty again.
+- Write the `Bet` blueprint id again
+- Click "Confirm" and you should see the next screen for initialization
+
+# Initialization
+### Address to Sign
+- Press "Create" without inserting any data: all fields should turn red
+- Insert any invalid data into "Address to Sign" and press "Create". Validation should still be red
+- Insert a valid address that is not from this wallet and press "Create". The validation persists.
+- Paste an address from this wallet that you already know. The validation should pass with green color.
+- Click "Select from a list" and select one of the addresses there. Validation still green.
+
+### Oracle Script
+- Insert "s" into this field. It should still be red.
+- Insert "a". The field should turn green immediately, without need to press the "Create" button.
+- Insert a valid script like `76a914db6fe378a8af070b332104c66c0a83dcb2d03e8b88ac`.
```

Comment:
It's important to share here the code to get to this script value. Otherwise the QA runner won't be able to set the bet result

Reply:
Good point. Added the code as an annex to the QA.